app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    read_file = command.split()[1]
+    write_file = command.split()[2]
+    if read_file != write_file and command.split()[0] == "cp":
```

Comment:
Perform a `split` operation once and write it to a variable, after that check the length of the variable, and in case its length is 3 use unpacking and later use obtained variables in the logic. Do not retrieve values index here